doc/source/whatsnew/v0.25.0.rst

```diff
@@ -36,6 +36,8 @@ Other Enhancements
 - :class:`RangeIndex` has gained :attr:`~RangeIndex.start`, :attr:`~RangeIndex.stop`, and :attr:`~RangeIndex.step` attributes (:issue:`25710`)
 - :class:`datetime.timezone` objects are now supported as arguments to timezone methods and constructors (:issue:`25065`)
 - :meth:`DataFrame.query` and :meth:`DataFrame.eval` now supports quoting column names with backticks to refer to names with spaces (:issue:`6508`)
+- :meth:`DatetimeIndex.intersection` and :meth:`TimedeltaIndex.intersection` have been moved to :meth:`Datetimelike.intersection` and tests were added for :meth:`TimedeltaIndex.intersection` (:issue:`24966`).
+- :meth:`TimedeltaIndex.intersection` now also supports the ``sort`` keyword.
```

Comment:
* No need to mention that tests were added.
* Is there an issue number for the "sort" bullet point?  Otherwise, reference the PR as the "issue".

Reply:
Took out tests mention and added the issue number for both.